sunpy/image/coalignment.py

```diff
@@ -287,6 +288,25 @@ def parabolic_turning_point(y):
     return numerator / denominator
 
 
+def check_for_nonfinite_entries(layer_image, template_image):
+    """
+    Issue a warning if there is any nonfinite entry in the layer or template images.
+
+    Parameters
+    ----------
+    layer_image : `numpy.ndarray`
+        A two-dimensional `numpy.ndarray`.
+    template_image : `numpy.ndarray`
+        A two-dimensional `numpy.ndarray`.
+    """
+    if not np.all(np.isfinite(layer_image)):
+        warnings.warn('The layer image has noninfinite entries.', SunpyUserWarning)
+
+    if not np.all(np.isfinite(template_image)):
+        warnings.warn('The template image has noninfinite entries.', SunpyUserWarning)
+
+
+@deprecated("1.1")
```

Comment:
I think we will have to wait til 2.1 to deprecate.

Reply:
We can deprecate now, just wont remove till 2.1

Reply:
Ah ok. Clearly I'm still a bit confused about what the policy is 😄